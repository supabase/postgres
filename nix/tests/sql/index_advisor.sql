create schema v;

create table v.book(
  id int primary key,
  title text not null
);

select
  index_statements, errors
from
  index_advisor('select id from v.book where title = $1');

drop schema v cascade;
