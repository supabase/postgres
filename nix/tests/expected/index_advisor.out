create schema v;
create table v.book(
  id int primary key,
  title text not null
);
select
  index_statements, errors
from
  index_advisor('select id from v.book where title = $1');
                index_statements                | errors 
------------------------------------------------+--------
 {"CREATE INDEX ON v.book USING btree (title)"} | {}
(1 row)

drop schema v cascade;
NOTICE:  drop cascades to table v.book
